src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -187,4 +193,13 @@ OAuthToken requestOAuthToken(String code) {
                 .bodyToMono(OAuthToken.class);
         return postReq.block(timeoutDuration);
     }
+
+    private void rengenrateSessionID(HttpServletResponse httpServletResponse) {
```

Comment:
Spelling and casing error, `rengenrateSessionID` should be `regenrateSessionId`.